.gitmodules

```diff
@@ -1,4 +1,9 @@
 [submodule "lmdb"]
-	path = lmdb
-	url = https://git.openldap.org/openldap/openldap.git
-        branch = LMDB_0.9.33
+  path = lmdb
+  url = https://github.com/LMDB/lmdb
```

Comment:
GitHub maintains an auto-synced 1:1 mirror of the upstream `openldap.git` here: https://github.com/openldap/openldap
That also contains the exact tag that this was previously using: https://github.com/openldap/openldap/releases/tag/LMDB_0.9.33